src/components/Form.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+
+type propsType= {
+    onSubmit(newToDo:{title: string, completed: boolean}):void;
+}
+
+function Form(props: propsType){
+
+    const [value, setValue] = useState<string>("")
+
+    function handleSubmit(e:any){
+        e.preventDefault();
+        if(value !== "" && !value.startsWith(" ")){
```

Comment:
```suggestion
        if(value.trim() !== "" ){
```

your code will onyl test fro one space!!!